types/index.d.ts

```diff
@@ -1,12 +1,13 @@
-import { DocumentNode } from 'graphql';
+import { DocumentNode } from "graphql";
+import { TypedDocumentNode } from "@graphql-typed-document-node/core";
 
 export type GqlRequest = <Result = {}, Variables = {}>(
-  gqlTag: DocumentNode,
+  gqlTag: DocumentNode | TypedDocumentNode<Result, Variables>,
```

Comment:
Can you describe the motivation for these changes, please?
what issue are you trying to solve?

Reply:
within a couple of weeks, the package with your changes will be updated, this is due to the fact that it is planned to add types for triggers events (because `FunctionEvent` is not an exact type for working with triggers)